### PR TITLE
Create JCB templates for marine B-matrix

### PR DESCRIPTION
**Pull request by DavidNew-NOAA**
This PR is a companion to Global Workflow PR [#2992](https://github.com/NOAA-EMC/global-workflow/pull/2992) and GDASApp PR [#1346](https://github.com/NOAA-EMC/GDASApp/pull/1346).

It adds JCB templates for the input YAMLS of JEDI applications used in the marine B-matrix task in the Global Workflow.

### CODE REVIEW

**Review by danholdaway — COMMENTED** [review]
Looks good. It might be good to modify the testing of marine in the main JCB repo to include these algorithms. There's an example of that for the atmosphere.

**Review by guillaumevernieres — COMMENTED** [review]
The 2 diffusion yamls are missing the "levels" key. I thought I modified in a previous prit but maybe not. I'll look into it later today.

**Review by danholdaway — APPROVED** [review]

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
Some updates are needed to the b-mat yaml, I'll look into it today.

File: algorithm/marine/soca_parameters_diffusion_vt.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    geometry: *geom
+    calibration:
+      normalization:
+        # NOTE, not actually used here, since the normalization spec is only used for hz
+        method: randomization  #< other option is "brute force"
+        iterations: 1000       #< in the real world you'll want to use 1e4 or so
+
+      groups:
+      - vertical:
+          as gaussian: true
```
Comment (guillaumevernieres) [inline algorithm/marine/soca_parameters_diffusion_vt.yaml.j2:26]:
```suggestion
          as gaussian: true
          levels: '{{ marine_vt_levels }}'
```

**Comment by guillaumevernieres** [comment]
Sorry for the late review @DavidNew-NOAA . I went through the yamls and it's only a one liner change.

File: algorithm/marine/soca_parameters_diffusion_vt.yaml.j2 (hunk 1 above)
Comment (danholdaway) [reply algorithm/marine/soca_parameters_diffusion_vt.yaml.j2:26]: This change would pass now.

**Comment by DavidNew-NOAA** [comment]
> Looks good. It might be good to modify the testing of marine in the main JCB repo to include these algorithms. There's an example of that for the atmosphere.

@danholdaway Can you point me to where specifically? Is these testing changes necessary before this PR is merged?

**Review by guillaumevernieres — APPROVED** [review]

**Comment by danholdaway** [comment]
I'll merge this and you can add to the testing later on. You would add to [this](https://github.com/NOAA-EMC/jcb-gdas/blob/develop/test/client_integration/gdas-marine-templates.yaml) file with something that looks like [this](https://github.com/NOAA-EMC/jcb-gdas/blob/1e9d844b0225acc2a0bf913358b1cd8581f9a2a9/test/client_integration/gdas-atmosphere-templates.yaml#L8).

**Comment by DavidNew-NOAA** [comment]
Thanks @danholdaway

**Pull request merged**